### ISSUE TITLE
Error: Unknown config option "reporters"

### ISSUE THREAD

**Issue by TAGC** [abdulhannanali/jest-nyan-reporter#3]
I've followed the instructions in the README and added a `reporters` section to my `package.json`:

```json
...
  "jest": {
    "collectCoverageFrom": [
      "src/**/*.{js,jsx}"
    ],
    "setupFiles": [
      "<rootDir>/config/polyfills.js"
    ],
    "testPathIgnorePatterns": [
      "<rootDir>[/\\\\](build|docs|node_modules|scripts)[/\\\\]"
    ],
    "testEnvironment": "node",
    "testURL": "http://localhost",
    "transform": {
      "^.+\\.(js|jsx)$": "<rootDir>/node_modules/babel-jest",
      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
      "^(?!.*\\.(js|jsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
    },
    "transformIgnorePatterns": [
      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx)$"
    ],
    "moduleNameMapper": {
      "^react-native$": "react-native-web"
    },
    "reporters": [
      [
        "jest-nyan-reporter",
        {
          "suppressErrorReporter": false
        }
      ]
    ]
  },
...
```

When I try to run my tests, I get the following error:

```
yarn test v0.22.0
$ node scripts/test.js --env=jsdom --coverage
Error: Unknown config option "reporters" with value "jest-nyan-reporter,[object Object]". This is either a typing error or a user mistake and fixing it will remove this message.
```

**Comment by abdulhannanali** [comment]
Hey @TAGC I am afraid, the code for Custom Reporters hasn't been merged in the Jest main repo right now. You'll be able to use this reporter once it's done. Keeping this issue open.

**Comment by TAGC** [comment]
Custom reporters available as of Jest 20.0.0. Issue resolved.
